### DISCUSSION TITLE
Format of `original_language` in `config.ini`

### DISCUSSION THREAD

**Discussion by Susko3** [ThioJoe/Auto-Synced-Translated-Dubs#1] [Q&A] [answered]
What's the format / supported languages for the `original_language` field in `config.ini`? Unlike all the other options, this one doesn't have a comment explaining it...

https://github.com/ThioJoe/Auto-Synced-Translated-Dubs/blob/b22916a3aa24eda0fb52237adbaff51d98ee3822/config.ini#L10

**Comment by ThioJoe** [comment, marked as answer]
You can see a list here and [here](https://learn.microsoft.com/en-us/azure/cognitive-services/speech-service/language-support?tabs=stt-tts) where Microsoft lists their TTS voices, it's the left most column. The list from Google is [here](https://cloud.google.com/text-to-speech/docs/voices). They're standardized so both lists should have the same codes per language
